docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -4,94 +4,83 @@ By [Phillip Carter](https://github.com/cartermp)
-* [How to target a Portable Class Library (PCL)](#how to target a portable class-library-pcl)
-* [How to cross-compile for .NET Core and .NET Framework](#how-to-cross-compile-for-net-core-and-net-framework)
+* [How to target the .NET Framework](#how-to-target-the-net-framework)
+* [How to target a Portable Class Library (PCL)](#how-to-target-a-portable-class-library-pcl)
+* [How to multitarget](#how-to-multitarget)
+* [How to use native dependencies](#how-to-use-native-dependencies)
 * [How to test libraries on .NET Core](#how-to-test-libraries-on-net-core)
+* [How to use multiple projects](#how-to-use-multiple-projects)
 * [How to create a NuGet Package with your Library](#how-to-create-a-nuget-package-with-your-library)
 
 ## Prerequisites
 
-You must have .NET Core installed on your machine.  You have two options:
+You must have .NET Core installed on your machine.  You will need [the .NET Core SDK and CLI](http://dotnet.github.io/getting-started/).
 
-* [DNX](http://aspnet.readthedocs.org/en/latest/getting-started/index.html), which is part of ASP.NET 5 RC1 and is included in Visual Studio Update 1.
-
-* [.NET CLI](http://dotnet.github.io/getting-started/), which will be used by .NET Core and ASP.NET RC2 and will replace DNX.
-
-Either toolchain can be used to build and publish libraries today.
-
-The sections of this document dealing with .NET Framework versions or Portable Class Libraries (PCLs) need the .NET Framework installed.  This is only supported on Windows.
-
-To do this, [install the .NET Framework](http://getdotnet.azurewebsites.net).
+The sections of this document dealing with the .NET Framework versions or Portable Class Libraries (PCLs) need the .NET Framework installed.  They are only supported on Windows.  To do this, [install the .NET Framework](http://getdotnet.azurewebsites.net).
 
 Additionally, if you wish to support older targets, you will need to install targeting/developer packs for older framework versions from the [target platforms page](http://getdotnet.azurewebsites.net/target-dotnet-platforms.html).  Refer to this table:
 
-| .NET Framework Version | Thing to download |
+| .NET Framework Version | What to download |
 | ---------------------- | ----------------- |
 | 4.6 | .NET Framework 4.6 Targeting Pack |
 | 4.5.2 | .NET Framework 4.5.2 Developer Pack |
 | 4.5.1 | .NET Framework 4.5.1 Developer Pack |
 | 4.5 | Windows Software Development Kit for Windows 8 |
 | 4.0 | Windows SDK for Windows 7 and .NET Framework 4 |
-| 2.0 and 3.5 | .NET Framework 3.5 SP1 Runtime (or Windows 8+ version) |
+| 2.0, 3.0, and 3.5 | .NET Framework 3.5 SP1 Runtime (or Windows 8+ version) |
 
 ## How to target .NET Core
 
 For the purposes of building a library, targeting ".NET Core" means targeting the .NET Platform Standard.  To see what that means exactly, see [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md).
```

Comment:
the .net platform standard topic has moved to a new location apparently: https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md.